musit-models/src/main/scala/no/uio/musit/models/Module.scala

```diff
@@ -0,0 +1,25 @@
+package no.uio.musit.models
+
+import play.api.libs.json.{JsNumber, Reads, Writes, __}
+
+sealed trait Module {
```

Comment:
Can we rename this to GroupModule or MusitModule (or so) to not get name conflicts with for instance guice modules.